sample_building/sample_building.py

```diff
@@ -516,12 +520,14 @@ def azimuth_to_antenna_offset(beamdir, main_antenna_count, interferometer_antenn
         phase_array = []
         for channel in range(0, main_antenna_count):
             # Get phase shifts for all channels
+            # zero pulse shift b/w pulses when beamforming.
             phase_array.append(get_phshift(beam, freq, channel, 0, main_antenna_count,
                 main_antenna_spacing))
-        for channel in range(0, interferometer_antenna_count):  # interferometer TODO interferometer offset ***
-            # Get phase shifts for all channels
+        for channel in range(0, interferometer_antenna_count): 
+            # Get phase shifts for all channels, adding in the x - offset of the interferometer
+            # from the main array.
             phase_array.append(get_phshift(beam, freq, channel, 0, interferometer_antenna_count,
-                interferometer_antenna_spacing))  # zero pulse shift b/w pulses when beamforming.
+                interferometer_antenna_spacing, centre_offset=intf_offset[0]))  
```

Comment:
Can intf_offset be a dict instead? This would easier to get at first glance if it was intf_offset['x']